activiti-spring-boot/spring-boot-starters/activiti-spring-boot-starter-hal-rest-api/src/main/java/org/activiti/client/model/ProcessDefinition.java

```diff
@@ -26,18 +28,43 @@
     private String name;
     private String description;
     private int version;
+    private List<String> users;
+    private List<String> groups;
+    private List<String []> variables;
+    private List<String []> userTasks;
+    private List<String []> serviceTasks;
 
```

Comment:
Having a separate class means that we are dealing with this as a sub resource inside the process-definitions/ resource. I don't want to pollute the ProcessDefinition type with this data that might be considered for very specific cases only.